services/src/main/resources/db/changelog/lead-capture.db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,180 @@
+            <column name="LC_LEAD_ID" type="BIGINT">
+                <constraints nullable="false" foreignKeyName="FK_LC_LEAD" references="ADDONS_LC_LEAD(ID)"/>
+            </column>
+            <column name="CREATED_DATE" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="ADDONS_LC_FORM_FIELD">
+            <column name="ID" type="BIGINT" autoIncrement="${autoIncrement}" startWith="1">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_ADDONS_LC_FORM_FIELD"/>
+            </column>
+            <column name="LC_RESPONSE_ID" type="BIGINT">
+                <constraints nullable="false" foreignKeyName="FK_LC_RESPONSE" references="ADDONS_LC_RESPONSE(ID)"/>
+            </column>
+            <column name="NAME" type="NVARCHAR(250)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="VALUE" type="NVARCHAR(250)">
+            </column>
+        </createTable>
+
+        <createTable tableName="ADDONS_LC_MAIL_TEMPLATE">
+
+            <column name="ID" type="BIGINT" autoIncrement="${autoIncrement}" startWith="1">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_LC_MAIL_TEMPLATE"/>
+            </column>
+            <column name="NAME" type="NVARCHAR(250)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="DESCRIPTION" type="NVARCHAR(250)">
+            </column>
+            <column name="EVENT" type="NVARCHAR(100)">
+            </column>
+            <column name="FORM" type="NVARCHAR(100)">
+            </column>
+            <column name="FIELD" type="NVARCHAR(100)">
+            </column>
+
+        </createTable>
+
+        <createTable tableName="ADDONS_LC_MAIL_CONTENT">
+
+            <column name="ID" type="BIGINT" autoIncrement="${autoIncrement}" startWith="1">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_LC_MAIL_CONTENT"/>
+            </column>
+            <column name="LANGUAGE" type="NVARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="SUBJECT" type="LONGTEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CONTENT" type="LONGTEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="LC_MAIL_TEMPLATE_ID" type="BIGINT">
+                <constraints nullable="false" foreignKeyName="FK_LC_MAIL_TEMPLATE"
+                             references="ADDONS_LC_MAIL_TEMPLATE(ID)"/>
+            </column>
+        </createTable>
+
+        <modifySql dbms="mysql">
+            <append value=" ENGINE=INNODB CHARSET=UTF8 COLLATE utf8_general_ci"/>
+        </modifySql>
+
+    </changeSet>
+
+    <changeSet author="exo-lead-capture" id="1.0.0-2" dbms="oracle,postgresql">
```

Comment:
Is the Oracle compatibility really necessary ?

Reply:
No, we need only postgres compatibility .